.github/workflows/wordpress-plugin-deploy.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       run: |
         npm install
         npm run build
+        composer install --no-dev
```

Comment:
do we need to add this same step to the readme/asset action as well?
https://github.com/10up/sophi-debug-bar/blob/5f13d331ff27a21802d62188d6561f93ad44b14c/.github/workflows/wordpress-plugin-asset-update.yml#L12-L15

Reply:
Correct, we also need vendor folder to exist (otherwise the deploy script might consider "Other files have been modified")